### ISSUE TITLE
IndexError during MLST

### ISSUE THREAD

**Issue by PHemarajata** [bactopia/bactopia#275]
Hi! I was trying to process paired-end reads for a Vibrio cholerae isolate. I've already downloaded species-specific dataset and was able to start the workflow but ran into an error during MLST. I checked mlst-blast.stderr.txt and it contained error log below.
```
Traceback (most recent call last):
  File "/home/peerahemarajata/miniconda3/envs/bactopia/share/bactopia-2.0.x/bin/mlst-blast.py", line 184, in <module>
    blast_alleles(args.fasta, args.blast, args.output, str(args.cpu),
  File "/home/peerahemarajata/miniconda3/envs/bactopia/share/bactopia-2.0.x/bin/mlst-blast.py", line 64, in blast_alleles
    alleles.append(f'{name}.{cols[i]}')
IndexError: list index out of range
```
I was just wondering if I did anything wrong. I am not sure what information you would need in order to help me with this issue so please let me know! Thank you so much for making such a great tool!

**Comment by rpetit3** [comment]
Hi @PHemarajata!

I think you found a fun schema. The error suggests one of the lines does not have the expected number of columns.

I'm going to look into this (https://pubmlst.org/bigsdb?db=pubmlst_vcholerae_seqdef&page=downloadProfiles&scheme_id=1) and update soon.

Cheers,
Robert

**Comment by rpetit3** [comment]
Found the issue! There's an extra column that wasn't accounted for `species`

```
ST      gyrB    pyrH    recA    atpA    species
1       1       1       1       1
2       2       2       2       2
3       3       3       3       3
4       4       4       4       4
```

I'm going fix this so that the extra columns don't cause issues in the future.

Thanks for finding this!

**Comment by rpetit3** [comment]
Alright, I've submitted a fix for this: https://github.com/bactopia/bactopia/commit/e0566286bb860e66f1b0da7f3245be2dd9c9005e

It'll be available in the next version

**Comment by PHemarajata** [comment]
Thank you!

**Comment by rpetit3** [comment]
Thank you for your help with this issue! I've released v2.0.2 (https://github.com/bactopia/bactopia/releases/tag/v2.0.2) which includes a fix for this issue, but also many other changes! I'm going to go ahead and close this for now, please feel free to reopen!

If you have a moment, could you take a few minutes to fill out a [Bactopia Usage Survey](https://forms.gle/PTgUxEBwCsCdbQjr7)? It really helps me direct the direction of Bactopia's development!
